docs/openshift/deploy.md

```diff
@@ -117,3 +117,53 @@ Run `acs-engine generate examples/openshift.json`
 After couple of minutes, your OpenShift web console should be accessible at `https://${dnsprefix}.${location}.cloudapp.azure.com:8443/`. You can log in with `clusterUsername` and `clusterPassword` values you set in the `openshift.json` file or with your Azure account if you've used AAD integration.
 
 For next steps, see [getting started documentation](https://docs.openshift.org/latest/getting_started/index.html) on OpenShift website.
+
+## Custom VNET
+
+ACS Engine supports deploying into an existing VNET. Operators must specify the ARM path/id of Subnets for the `masterProfile` and  any `agentPoolProfiles`, as well as the first IP address to use for IP static IP allocation in `firstConsecutiveStaticIP`. Additionally, to prevent source address NAT'ing within the VNET, we assign to the `vnetCidr` property in `masterProfile` the CIDR block that represents the usable address space in the existing VNET.
```

Comment:
`to use for IP static IP allocation` - extra IP in there?

Reply:
@pweil- don't understand your comment.  I'd suggest
s/the first IP address to use for IP static IP allocation/the master IP address/

Reply:
> as well as the first IP address to use for IP static IP allocation

as well as the first IP address to use for static IP allocation in ...

^ that read better to me, I'm ok with saying it's specifically for the master though too

Reply:
@jim-minter we had to specify `vnetCidr`. without it, it will use `10.0.0.0/24` as set by `DefaultOpenShiftMasterSubnet` and the deployment will end in an error complaining `firstConsecutiveStaticIP` is not in `10.0.0.0/24` address space.

Reply:
I was able to stand up a cluster in the 10.239.0.0 range following your document, without specifying vnetCidr.  By inspection I can't see any place where vnetCidr would influence an OpenShift setup - I think it's a no-op.  I can see it's necessary for Kubernetes, but the networking mechanisms are sufficiently different between the two platforms that I can't see why vnetCidr would be needed for OpenShift.

Reply:
Please s/the first IP address to use for IP static IP allocation/the master IP address/.